project/deps.sc

```diff
@@ -62,12 +62,13 @@ object Deps {
   def slf4jNop                 = ivy"org.slf4j:slf4j-nop:1.8.0-beta4"
   def sparkSql                 = ivy"org.apache.spark::spark-sql:2.4.0"
   def utest                    = ivy"com.lihaoyi::utest:0.7.10"
+  def os_lib                   = ivy"com.lihaoyi::os-lib:0.8.0"
 }
 
 object ScalaVersions {
-  def scala3   = "3.0.1"
-  def cross2_3Version = "2.13.6"
-  def scala213 = "2.13.6"
+  def scala3   = "3.0.2"
+  def cross2_3Version = "2.13.7"
+  def scala213 = "2.13.7"
```

Comment:
Besides changing `deps.sc`, you probably need to modify GitHub Actions too https://github.com/almond-sh/almond/blob/master/.github/workflows/ci.yml#L49

Reply:
Oooh - thankyou for your help on my two random pull requests... I think these changes have solved the actual compatibility issues... now apparently the website grumbles :-)!